scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -19,7 +19,7 @@ workflow GvsImportGenomes {
     Int batch_size = 1
 
     Int? preemptible_tries
-    File? gatk_override
+    File? gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/ah_var_store_20210914/gatk-package-4.2.0.0-406-ga9206a2-SNAPSHOT-local.jar"
```

Comment:
is there any benefit to making this a constant somewhere so we dont have to update it in multiple places? ( I guess it's only 2, so not really an issue though)

Reply:
yeah and I don't know of a good way to have a constant/shared value across two WDLs